src/java/com/github/jsdossier/CompilerModule.java

```diff
@@ -78,7 +79,7 @@ CompilerOptions provideCompilerOptions(
 
     // IDE mode must be enabled or all of the jsdoc info will be stripped from the AST.
     options.setIdeMode(true);
```

Comment:
I'd guess that the rest of IDE mode isn't needed and can be removed. But you probably know what js-dossier needs better than I do.